### DISCUSSION TITLE
Lightning Terminal's Metric Ratings:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightninglabs/lightning-terminal#757] [Q&A]
Hi,  Can you please tell me how Lightning terminal comes up with these ratings?  We see on our node: Handshake that our rating is 1 good Inboud Peer to 10 good outbound peers. We wish to have a lot more good inbound peers so as to increase our routing flow, etc...  Can you please tell us how we can achieve this, how to we get more than 1 good inbound peer because as wel all know how important it is to have good inbound peers because without them nothing will flow in and of course nothing will also flow out. Thank you :)

**Comment by levmi** [comment]
We can't tell you exactly how the Rankings are constructed because of our belief in Goodhart's Law, an adage that states "When a measure becomes a target, it ceases to be a good measure." In other words, if we publicly declared exactly how we achieved the rankings, people would game the outcome rather than just trying to be a best in class routing node. With that said, we do have an article in our [Builder's Guide about the Rankings](https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/ranking) specifically. As for your specific question on getting additional inbound peers, you should look for nodes with Easy to Reach high percentile Inbound Quality scores on the [Rankings](https://terminal.lightning.engineering/) and then peer with a few of those nodes.

**Comment by BonzoDogDooDah** [comment]
Hi Michael, Thank you for this useful information. If you don't mind I have another question for you please. Just to confirm this statement; "Easy to Reach high percentile Inbound Quality scores on the Rankings <https://terminal.lightning.engineering/>" is referring to this; Easy to Reach (highest 60 percentile): and any node with this ranking is considered to be a node we should consider creating a channel with to add more good inbound routing peers to our node?

Thank you for your assistance :)

BonzoDogDooDah


-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


May 9, 2024, 13:41 by ***@***.***:

**Reply by BonzoDogDooDah** [reply]
Hi Sorry for that question I believe I have answered it myself by deciding to use my own brain and to not bother you. Thank you. :)
